local_devnet/celestia-app/app.toml

```diff
@@ -43,7 +43,7 @@ halt-time = 0
 # It has no bearing on application state pruning which is determined by the
 # "pruning-*" configurations.
 #
-# Note: Tendermint block pruning is dependant on this parameter in conjunction
+# Note: Tendermint block pruning is dependent on this parameter in conjunction
```

Comment:
The previous spelling is fine for British English: https://www.merriam-webster.com/grammar/spelling-variants-dependent-vs-dependant#:~:text=To%20be%20safe%2C%20use%20'dependant,...%20and%20in%20Britain.&text=The%20difference%20between%20dependent%20and,'